architecture/bela.cpp

```diff
@@ -693,11 +976,55 @@ class TrillCraftWidget : public TrillWidget
                     }
                     *fZone = val;
                 }
+#ifdef NVOICES 
```

Comment:
Is conditional compilation really needs here ?

Reply:
no